docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -136,6 +136,25 @@ General-purpose aggregation functions
 
         Examples are the same as for :attr:`ArrayAgg.ordering`.
 
+    Usage example::
+
+        class Person(models.Model):
+            info = models.JSONField()
+
+        >>> from django.contrib.postgres.aggregates import JSONBAgg
+        >>> Person.objects.create(info={'name': 'foo'})
+        >>> Person.objects.aggregate(jsonbagg=JSONAgg('info'))
+        {'jsonbagg': [{'name': 'mariyam'}]}
+
+    When used to aggregate on a :class:`~django.db.models.JSONField`,
+    :lookup:`key or index based transformations <jsonfield.key>` can be
+    chained with it.
+
+        >>> Person.objects.annotate(jsonbagg=JSONBAgg('info')).filter(jsonbagg__0__name='foo').values('jsonbagg')
+        <QuerySet [{'jsonbagg': [{'name': 'mariyam'}]}]>
```

Comment:
We should use more realistic example, e.g. with related fields (the same in tests):
```python
    def test_jsonb_agg_key_index_transforms(self):
        room101 = Room.objects.create(number=101)
        room102 = Room.objects.create(number=102)
        datetimes = [
            timezone.datetime(2018, 6, 20),
            timezone.datetime(2018, 6, 24),
            timezone.datetime(2018, 6, 28),
        ]   
        HotelReservation.objects.create(
            datespan=(datetimes[0].date(), datetimes[1].date()),
            start=datetimes[0],
            end=datetimes[1],
            room=room102,
            requirements={'double_bed': True},
        )   
        HotelReservation.objects.create(
            datespan=(datetimes[1].date(), datetimes[2].date()),
            start=datetimes[1],
            end=datetimes[2],
            room=room102,
            requirements={'double_bed': False, 'smoking': True},
        )   
        HotelReservation.objects.create(
            datespan=(datetimes[0].date(), datetimes[2].date()),
            start=datetimes[0],
            end=datetimes[2],
            room=room101,
            requirements={'smoking': False},
        )
        values = Room.objects.annotate(
            requirements=JSONBAgg(
                'hotelreservation__requirements',
                ordering='-hotelreservation__start',
            )   
        ).filter(requirements__0__smoking=True).values('number', 'requirements')
        self.assertSequenceEqual(values, [
            {'number': 102, 'requirements': [
                {'double_bed': False, 'smoking': True},
                {'double_bed': True}, 
            ]}, 
        ])
```

Reply:
Thanks, updated.